src/pages/events/create.tsx

```diff
@@ -0,0 +1,202 @@
+              <Input
+                type="text"
+                disabled={isSubmitting}
+                {...register('eventName', { required: true })}
+              />
+              {errors.eventName && (
+                <Text color="tomato" className="pt-2">
+                  {errors.eventName.message}
+                </Text>
+              )}
+            </div>
+            <VStack align="left">
+              <div className="flex">
+                <FormLabel>Department</FormLabel>
+                <CheckboxGroup>
+                  <Stack spacing={[1, 5]} direction={['row', 'column']}>
+                    <Checkbox value="ml" {...register('dept')}>
+                      Machine Learning
+                    </Checkbox>
+                    <Checkbox value="sd" {...register('dept')}>
+                      Software Development
+                    </Checkbox>
+                    <Checkbox value="bc" {...register('dept')}>
+                      Blockchain
+                    </Checkbox>
+                    <Checkbox value="ir" {...register('dept')}>
+                      Internal Relations
+                    </Checkbox>
+                    <Checkbox value="ea" {...register('dept')}>
+                      External Affairs
+                    </Checkbox>
+                  </Stack>
+                </CheckboxGroup>
+              </div>
+              {errors.dept && (
+                <Text color="tomato" className="pt-2">
+                  {errors.dept.message}
+                </Text>
+              )}
+            </VStack>
+            <div>
+              <FormLabel>Date</FormLabel>
+              <Input
+                placeholder="Select Date and Time"
+                size="md"
+                type="datetime-local"
+                disabled={isSubmitting}
+                className="dark:[color-scheme:dark]"
+                {...register('date', { required: true })}
+              />
+              {errors.date && (
+                <Text color="tomato" className="pt-2">
+                  {errors.date.message}
+                </Text>
+              )}
+            </div>
+            <div className="flex items-center">
+              <FormLabel>QR Code required</FormLabel>
+              <Checkbox disabled={isSubmitting}></Checkbox>
+            </div>
+            <DataTable data={data} setAttendees={setAttendees} />
+            {submitBefore && invalidAttendees && (
+              <Text color="tomato">At least one attendee is required</Text>
+            )}
+            <div className="flex justify-between">
+              <Button
+                bgColor="#FF9900"
+                width={150}
+                textColor="black"
+                onClick={redirectHome}
+              >
+                Back
+              </Button>
+              <Button
+                bgColor="#4365DD"
+                width={150}
+                type="submit"
+                disabled={isSubmitting}
```

Comment:
- [ ] Is there any reason for the addition of the react-form dependency? The alternative would be `newEvent.isSubmitting`, `newEvent.error`, `newEvent.isSuccess` as of the TRPC Mutation Hook.

Reply:
Didn't realise that there was one for submission my bad will use TRPC mutation instead